### DISCUSSION TITLE
Checkout Page Blocks

### DISCUSSION THREAD

**Discussion by danelas** [woocommerce/woocommerce#53400] [Checkout Flow]
There should be an option to change font and most importantly a layout change, I would love for the product to be on the left and details on right. Stripe has a beautiful layout.
![Screenshot 2024-12-02 182909](https://github.com/user-attachments/assets/c0ce0e8f-7e68-41c7-97f3-2efcbd48f363)

**Comment by Drivingralle** [comment]
+1

**Comment by senadir** [comment]
Fonts comes from your theme, so if you customize those, they should come down to Checkout directly.

For layout, we're considering adding more layout options, for now, you can achieve this with simple CSS
